ci/release/update-version.sh

```diff
@@ -37,60 +38,83 @@ function sed_runner() {
 
 # Centralized version file update
 echo "${NEXT_FULL_TAG}" > VERSION
+echo "${NEXT_FULL_TAG}" > RAPIDS_VERSION
 
-dependencies='libcuopt libcuopt-cu12 cuopt cuopt-cu12 cuopt-mps-parser'
-for FILE in conda/environments/*.yaml dependencies.yaml; do
-    for dependency in ${dependencies}; do
-        sed_runner "s/\(${dependency}==\)[0-9]\+\.[0-9]\+/\1${PY_NEXT_SHORT_TAG}/" "${FILE}"
-    done
+DEPENDENCIES=(
+  libcuopt
+  libcuopt-cu12
+  cuopt-mps-parser
+  cuopt
+  cuopt-cu12
+  cuopt-server
+  cuopt-server-cu12
+  cuopt-sh-client
+  cudf
+  cudf-cu12
+  cuvs
+  cuvs-cu12
+  librmm
+  librmm-cu12
+  libraft-headers
+  rmm
+  rmm-cu12
+  pylibraft
+  pylibraft-cu12
+  raft-dask
+  raft-dask-cu12
+  rapids-dask-dependency
+)
```

Comment:
```suggestion
DEPENDENCIES=(
  cudf
  cuvs
  cuopt
  cuopt-mps-parser
  cuopt-server
  cuopt-sh-client
  libcuopt
  libraft-headers
  librmm
  pylibraft
  raft-dask
  rapids-dask-dependency
  rmm
)
```

Let's please alphabetize this (so it's easier to read at a glance) and remove `-cu12` entries (see next comment).